src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -46,4 +53,22 @@ public function run($request)
         }
         ImageThumbnailHelper::singleton()->run();
     }
+
+    /**
+     * TODO Refactor this whole mess into Symfony Console on a TaskRunner level,
+     * with a thin wrapper to show coloured console output via a browser:
+     * https://github.com/silverstripe/silverstripe-framework/issues/5542
+     * @throws \Exception
+     */
+    protected function addLogHandlers()
+    {
+        if ($logger = Injector::inst()->get(LoggerInterface::class)) {
+            if (Director::is_cli()) {
```

Comment:
might worth having a separate logger class implementation rather than encoding this logic in here